### PR TITLE
Dashboard Piecharts legend position/ hide- when entries are too many

### PR DESCRIPTION
**Pull request by mamy-CS**
https://github.com/spiffe/tornjak/issues/26
There is an issue open in carbon components to hide when legends are too many but it's not implemented yet. 
Temporary fix - position top entry legends on the right side. 
Signed-off-by: Mohammed Abdi <mohammed.munir.abdi@ibm.com>

### CODE REVIEW

File: tornjak-frontend/src/charts/PieChart.js [hunk 1]
```diff
@@ -10,10 +10,17 @@ class PieChart1 extends React.Component {
         resizable: true,
         height: "300px",
         legend: {
-          alignment: "center"
+          //enabled: false,
+          position: "right",
+          //alignment: "center",
```
Comment (lumjjb) [inline tornjak-frontend/src/charts/PieChart.js:15]: nit: remove dead code

**Review by lumjjb — COMMENTED** [review]
minor nits, else LGTM

File: tornjak-frontend/src/charts/PieChart.js (hunk 1 above)
Comment (mamy-CS) [reply tornjak-frontend/src/charts/PieChart.js:15]: done!

**Review by lumjjb — APPROVED** [review]
LGTM

**Pull request merged**